app/schema/portal/owner/owner-details.js

```diff
@@ -13,7 +13,7 @@ const validate = (value, helper) => {
     const today = startOfDay(new Date())
```

Comment:
Just a note for future - date-fns has an `isFuture` function if it's useful?

Reply:
Good to know